conversionSetup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#Installs all necessary apps to convert pdf to adoc
+# install asciidoctor
+
+sudo apt-get install asciidoctor
+
+asciidoctor -v
+
+
+# install pandoc
+
+sudo apt-get install pandoc
+
+
+# install python3.8
+
+sudo apt install python3.8
+
+
+# install python3.pip
+
+sudo apt-get -y install python3.pip
+
+
+# install pdf2txt
+
+pip3 install pdf2txt
+
+
+# install python3-pdfminer
+
+sudo apt install python3-pdfminer
```

Comment:
Are we actually using pdf miner?

Reply:
I do not believe we are, I kept this here because I have not tested the installation script without this line. Before removing it, I would like to try a conversionsetup.sh on a clean Ubuntu installation.

Reply:
validate this and once validated I'll approve this PR

Reply:
We are using pdfminer's pdf2txt functionality.

Reply:
ahhh so we do need this ok cool